aspnetcore/security/app-secrets.md

```diff
@@ -404,5 +404,6 @@ No secrets configured for this application.
 
 ## Additional resources
 
+* See [this issue](https://github.com/aspnet/AspNetCore.Docs/issues/16328) for information on accessing secret manager from IIS.
```

Comment:
```suggestion
* See [this issue](https://github.com/aspnet/AspNetCore.Docs/issues/16328) for information on accessing Secret Manager from IIS.
```

It might also be better to move this text into the Windows tab of the **How the Secret Manager tool works**. Only Windows folks would be interested in this info.